src/ensembl/src/shared/components/radio/Radio.tsx

```diff
@@ -14,6 +16,9 @@ type Props = {
   classNames?: {
     radioInput?: string;
     radioLabel?: string;
+    radioWrapper?: string;
+    defaultRadio?: string;
```

Comment:
Notice the repeating word "radio"? We could remove some of the repetition it by renaming the properties to:

- radio
- checked (or checkedRadio)
- label
- wrapper